### PR TITLE
Move End Shipper to general availability

### PR DESCRIPTION
**Pull request by nwithan8**
# Description

- Copy EndShipper class to main namespace
- Mark EndShipper beta class as deprecated
- Update EndShipper.all to return EndShipperCollection
- New EndShipperCollection class to wrap list of End Shipper objects
- Update unit tests
- Fix bad docstrings

# Testing

- Unit tests updated and passing.

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: src/main/java/com/easypost/model/AddressCollection.java [hunk 1]
```diff
@@ -11,7 +11,7 @@ public final class AddressCollection extends EasyPostResource {
     /**
      * Get a list of addresses.
      *
-     * @return List of Batch objects
+     * @return List of Address objects
```
Comment (Justintime50) [inline src/main/java/com/easypost/model/AddressCollection.java:14]: 🤦‍♂️

**Review by Justintime50 — DISMISSED** [review]

File: src/main/java/com/easypost/model/AddressCollection.java (hunk 1 above)
Comment (nwithan8) [reply src/main/java/com/easypost/model/AddressCollection.java:14]: Oh.
image

Comment (Justintime50) [reply src/main/java/com/easypost/model/AddressCollection.java:14]: Heheh 😂 Whoops

File: src/main/java/com/easypost/model/EndShipper.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+import com.easypost.exception.EasyPostException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class EndShipper extends BaseAddress {
+    /**
+     * Create EndShipper object from parameter map.
+     *
+     * @param params Map of end shipper parameters.
```
Comment (jchen293) [inline src/main/java/com/easypost/model/EndShipper.java:13]:
```suggestion
     * @param params Map of EndShipper parameters.
```
This also occurs in other docstrings, but won't block it

**Review by jchen293 — APPROVED** [review]

File: src/main/java/com/easypost/model/EndShipper.java (hunk 2 above)
Comment (nwithan8) [reply src/main/java/com/easypost/model/EndShipper.java]: Fixed, thanks for the callout

**Review by Justintime50 — APPROVED** [review]

**Review by jchen293 — APPROVED** [review]

**Pull request merged**